src/Dialog/formComponents/widgets/BaseInput.test.js

```diff
@@ -0,0 +1,46 @@
+/* global it, describe */
+import React from 'react';
+import chai from 'chai';
+import chaiEnzyme from 'chai-enzyme';
+import spies from 'chai-spies';
+import {shallow} from 'enzyme';
+
+import BaseInput from './BaseInput';
+
+chai.use(spies);
+chai.use(chaiEnzyme());
+chai.should();
+
+describe('< BaseInput />', function () {
+  it('should exist', () => {
+    const wrapper = shallow(
+      <BaseInput id={'0'} />
```

Comment:
If I don't pass an `id`, I find in test logs a warning that `BaseInput` needs an `id`. I think we don't want to have log like this ;)